client/post-editor/drafts-button/index.jsx

```diff
@@ -10,12 +10,12 @@ import { translate } from 'i18n-calypso';
  */
 import Button from 'components/button';
 import Count from 'components/count';
-import QueryPostCounts from 'components/data/query-post-counts';
+import QueryPosts from 'components/data/query-posts';
 import { getSelectedSiteId } from 'state/ui/selectors';
 import { isJetpackSite } from 'state/sites/selectors';
-import { getAllPostCount } from 'state/posts/counts/selectors';
```

Comment:
> We shouldn't count how many posts we have in state.posts because it may not represent the total 

I'm not sure I understand why the selector's count can not represent the right value since we use this same query to fetch the drafts to show.
--

BTW, you're saying  we stick to `getMyPostCounts` which I can understand but note that the drafts shown just under the button can contain more than "my drafts" only (The issue says it must be the same).

 I can update the Drafts List also to display only My Drafts, In this case, an "admin" user will find difficulties to see drafts from other authors.

I don't have all the necessary knowledge to make the "functional" choices yet :). Tks for your help.